DMX.Portal.Web.Tests.Unit/Components/NewLabDialogs/NewLabDialogComponentTests.Render.cs

```diff
@@ -0,0 +1,221 @@
+using Xunit;
+
+namespace DMX.Portal.Web.Tests.Unit.Components.NewLabDialogs
+{
+    public partial class NewLabDialogComponentTests
+    {
+        [Fact]
+        public void ShouldRenderDefaults()
+        {
+            // given . when
+            var initialNewLabDialogComponent = new NewLabDialog();
+
+            // then
+            initialNewLabDialogComponent.LabViewService.Should().BeNull();
+            initialNewLabDialogComponent.State.Should().Be(NewLabDialogComponentState.Loading);
+            initialNewLabDialogComponent.Dialog.Should().BeNull();
+            initialNewLabDialogComponent.LabId.Should().BeNull();
+            initialNewLabDialogComponent.LabDmxVersion.Should().BeNull();
+            initialNewLabDialogComponent.LabName.Should().BeNull();
+            initialNewLabDialogComponent.LabDescription.Should().BeNull();
+            initialNewLabDialogComponent.IsVisible.Should().BeFalse();
+            initialNewLabDialogComponent.LabView.Should().BeNull();
+            initialNewLabDialogComponent.Exception.Should().BeNull();
+            initialNewLabDialogComponent.ErrorMessage.Should().BeNull();
+            initialNewLabDialogComponent.Spinner.Should().BeNull();
+            initialNewLabDialogComponent.LabNameValidationSummary.Should().BeNull();
+            initialNewLabDialogComponent.LabDescriptionValidationSummary.Should().BeNull();
+            initialNewLabDialogComponent.LabIdContainer.Should().BeNull();
+            initialNewLabDialogComponent.LabDmxVersionContainer.Should().BeNull();
+            initialNewLabDialogComponent.LabNameContainer.Should().BeNull();
+            initialNewLabDialogComponent.LabDescriptionContainer.Should().BeNull();
+            initialNewLabDialogComponent.DependencyErrorMessageLabel.Should().BeNull();
+        }
+
+        [Fact]
+        public void ShouldDisplayDialogIfOpenDialogIsClicked()
+        {
+            // given
+            NewLabDialogComponentState expectedState =
+                NewLabDialogComponentState.Content;
+
+            var expectedLabView = new LabView
+            {
+                DmxVersion = "1.0"
+            };
+
+            string expectedTextBoxCssClass = "new-lab-textbox";
+            string expectedTextBoxBottomCssClass = "new-lab-textbox-bottom";
+            string expectedDependencyErrorMessageLabelCssClass = "new-lab-error-message";
+
+            // when 
+            this.renderedNewLabDialog = RenderComponent<NewLabDialog>();
+            this.renderedNewLabDialog.Instance.OpenDialog();
+
+            // then
+            this.renderedNewLabDialog.Instance.State.Should().Be(expectedState);
+            this.renderedNewLabDialog.Instance.LabViewService.Should().NotBeNull();
+            this.renderedNewLabDialog.Instance.Dialog.Should().NotBeNull();
+            this.renderedNewLabDialog.Instance.Dialog.IsVisible.Should().BeTrue();
+            this.renderedNewLabDialog.Instance.Dialog.Title.Should().Be("Lab Details");
```

Comment:
Create a variable and call it `expectedDialogTitle` - do the same for all the other hard coded values in your test.